demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -511,7 +511,7 @@ static int handlePublishResend( MQTTContext_t * pMqttContext )
      * packetId key and PublishPacket_t values may be used instead. */
     packetIdToResend = MQTT_PublishToResend( pMqttContext, &cursor );
 
-    while( packetIdToResend != 0 )
+    while( packetIdToResend != MQTT_PACKET_ID_INVALID )
```

Comment:
Can you make similar updates in other demos as well?

Reply:
was not done